pegasus/helpers/prize_helpers.rb

```diff
@@ -7,6 +11,7 @@ def claim_prize_code(type, email, params={})
   begin
     rows_updated = DB[:hoc_survey_prizes].where(claimant: nil, type: type).limit(1).update(
       claimant: email,
+      purpose: purpose,
       claimed_at: DateTime.now,
       claimed_ip: ip_address,
     )
```

Comment:
no because the prize is claimed asynchronously in process_forms (cron job)
We'll get an error logged in honeybadger and have to look into why the form failed to process, deal with it manually